src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -178,24 +178,27 @@
     "required": ["elements"],
 }
 
-element_types = ['issue-link', 'alert-rule-action', 'issue-media', 'stacktrace-link']
+element_types = ["issue-link", "alert-rule-action", "issue-media", "stacktrace-link"]
 
 
 def checkElementsIsArray(instance):
-    if not isinstance(instance['elements'], list):
+    if not isinstance(instance["elements"], list):
         raise SchemaValidationError("'elements' should be an array of objects")
 
 
 def checkForElementTypeError(instance):
-    for element in instance['elements']:
-        if 'type' not in element:
+    for element in instance["elements"]:
+        if "type" not in element:
             raise SchemaValidationError("Each element needs a 'type' field")
-        found_type = element['type']
-        if not found_type in element_types:
-            raise SchemaValidationError("Element has type '%s'. Type must be one of the following: %s" % (found_type, element_types))
+        found_type = element["type"]
+        if found_type not in element_types:
+            raise SchemaValidationError(
+                "Element has type '%s'. Type must be one of the following: %s"
+                % (found_type, element_types)
+            )
 
 
-def validate(instance):
+def validateUiElementSchema(instance):
```

Comment:
We generally use snake_case in python :snake:

Reply:
@markstory Yup, I realized that earlier and made the change. I think you're looking at an outdated version of the file.